src/Uno.UI.RuntimeTests/IntegrationTests/common/TestServices.KeyboardHelper.cs

```diff
@@ -118,7 +120,7 @@ public static class KeyboardHelper
 			};
 
 
-			public static void PressKeySequence(string keys, UIElement element = null)
+			public static async void PressKeySequence(string keys, UIElement element = null)
```

Comment:
Oh, callers should probably be updated to `await` the call then. Consider also renaming to `PressKeySequenceAsync`

```suggestion
			public static async Task PressKeySequence(string keys, UIElement element = null)
```

Reply:
I don't want to change the signature to stay compatible with WinUI tests, which have the same naming. They are also not awaiting, but they do expect the event to be raised asynchronously

Reply:
Currently this is fire-and-forget, so it should still be problematic

Reply:
I the same thing is happening in case of WinUI though - they also raise the event asynchronously in a "fire and forget" fashion and then just do some "waiting" action afterwards to know when the event was actually processed. The simple solution would be to adjust the API to be fully async, which is doable,, I was just trying to avoid it to keep the "portability" easier

Reply:
Then I would make this fire-and-forget from the beginning, ie, make `RaiseOnElementDispatcherAsync` non-async, and use `_ = element.Dispatcher.RunAsync(...)` instead of `await element.Dispatcher.RunAsync(...)`

Reply:
Had to keep the original approach, because otherwise we could theoretically schedule the events out of order, breaking the goal of pressing the given keys in a specific sequence...